site/docs/Contributing/before-contributing.mdx

```diff
@@ -54,4 +54,4 @@ Before you start working or proposing a new component, it is important to check
 Go through the list of existing [HDS components](/components). See if some of the components could be slightly altered to suit your needs while still fulfilling existing requirements. If yes, move on to making a proposal for the change.
 
 ### 3. If not, find out if is this component something that is needed in several projects?
-Think outside of the scope of your project. Can this component be utilized in other projects as well? If not, could you make it something more generic to match needs of other projects? Discuss with other projects and community.
\ No newline at end of file
+Think outside of the scope of your project. Can this component be utilized in other projects as well? If not, could you make it something more generic to match needs of other projects? Discuss with other projects and community.
```

Comment:
to match **the** needs of other projects? Discuss with other projects and **the** community.